### PR TITLE
Updated README.rst

### PR DESCRIPTION
**Pull request by jhamil64**
Added more clear titles to the README, and a helpful link to the "Hacking" wiki page. I would like to do more, but I don't want to step on anyone's feet and wait for feedback.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -24,7 +24,7 @@ Installation
    pip install pygame
 
 
-Help
+How to Use Pygame
```
Comment (novialriptide) [inline README.rst:27]: The original text was better imo.

**Review by novialriptide — COMMENTED** [review]
I like the rest, just not the one I just commented on

File: README.rst [hunk 2]
```diff
@@ -44,11 +44,11 @@ Best of all, the examples directory has many playable small programs
...
 
 
-Building From Source
+How to Contribute
 --------------------
 
-If you want to use features that are currently in development,
-or you want to contribute to pygame, you will need to build pygame
+If you want to contribute to pygame, or use features currently
+in development, you will need to build pygame
```
Comment (novialriptide) [inline README.rst:51]: It seems like the order is just switched?

File: README.rst [hunk 3]
```diff
@@ -44,11 +44,11 @@ Best of all, the examples directory has many playable small programs
 which can get you started playing with the code right away.
 
 
-Building From Source
+How to Contribute
```
Comment (novialriptide) [inline README.rst:47]: Some people might want to just build from source, so the original should stay.

**Comment by Starbuck5** [comment]
OP hasn't responded to feedback and it's been a few weeks, I'm going to close this PR.

If you'd like to try again with these changes you can open a new PR in the future.

**Pull request closed without merging**